src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -164,13 +160,43 @@ const ArtistSeriesHeaderSmall: React.FC<ArtistSeriesHeaderProps> = props => {
   )
 }
 
+const StyledGrid = styled(Grid)`
+  @media (max-width: ${theme.flexboxgrid.breakpoints.lg - 1}px) {
```

Comment:
The default styling of `Grid` causes white spacing issues at smaller viewports with this design, resulting into too much white space between the header image and the page margins. So, for small and medium-sized viewports we allow the grid to expand to the full width of the container so that the text and image and can take up the whole space like in the design.

Reply:
What is the `-1` for?

Reply:
That's to set the `max-width` to the largest screen size below large.